cellfinder/napari/train/train.py

```diff
@@ -142,12 +190,24 @@ def widget(
         if yaml_files[0] == Path.home():  # type: ignore
             show_info("Please select a YAML file for training")
         else:
-            worker = run_training(
+            progress_bar.visible = True
+            progress_bar.label = "Initializing training..."
+            progress_bar.max = 1
+            progress_bar.value = 0
+
+            worker = TrainingWorker(
                 training_data_inputs,
                 optional_network_inputs,
                 optional_training_inputs,
                 misc_training_inputs,
             )
+            worker.connect_progress_bar_callback(progress_bar)
+            worker.finished.connect(
+                lambda: setattr(progress_bar, "visible", False)
```

Comment:
```suggestion
                lambda: progress_bar.setVisible(False)
```
I _think_ this works, and reads nicer?

Reply:
Great suggestion! 🙌 While `progress_bar.setVisible(False)` would have been perfect, I found that `magicgui.widgets.ProgressBar` doesn't support it directly. Instead, `setattr(progress_bar, "visible", False)` did the trick. Thanks for the input